config/routes.rb

```diff
@@ -0,0 +1,13 @@
+Rails.application.routes.draw do
+  root 'root#index'
+  namespace :api do
+    resources :messages do
+      get 'random_greeting', on: :collection
+    end
+  end
+
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+end
```

Comment:
- [ ] I can see that you have not applied this particular change made by the previous reviewer. Please kindly apply that.